src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/ReadsDataflowSource.java

```diff
@@ -143,6 +149,25 @@ public SAMFileHeader getHeader() {
     }
 
     public static PCollectionView<SAMFileHeader> getHeaderView(Pipeline p, SAMFileHeader readsHeader) {
-        return p.apply(Create.of(readsHeader).withCoder(SerializableCoder.of(SAMFileHeader.class))).apply(View.<SAMFileHeader>asSingleton());
+        return p.apply(Create.of(readsHeader).withCoder(SerializableCoder.of(SAMFileHeader.class))).setName("reads header").apply(View.<SAMFileHeader>asSingleton());
     }
+
+
+
+    /**
+     * Create a {@link PCollection< KV <String,Iterable<GATKRead>>>} containing all the reads starting in the given intervals and their mates,
+     * grouped by the shard they start in.
+     * The shards are strings as per SimpleInterval's ToString syntax.
```

Comment:
it's also in hellender README - dont use toString for things that are not human consumption